crates/nargo_cli/src/cli/compile_cmd.rs

```diff
@@ -38,6 +41,10 @@ pub(crate) struct CompileCommand {
     #[arg(long)]
     include_keys: bool,
 
+    /// Output debug files
+    #[arg(long)]
+    output_debug: bool,
```

Comment:
```suggestion
    /// Output debug files
    #[arg(long, hide = true)]
    output_debug: bool,
```

There's a good chance that we're going to be making a bunch of breaking changes to debug info in the near future and this is only really needed by A3 atm so we shouldn't make this a fully supported feature.